types/ember__object/events.d.ts

```diff
@@ -1,34 +1,32 @@
 import { ObserverMethod } from '@ember/object/-private/types';
 
+export {};
```

Comment:
This package doesn't export an empty object in reality, so we can't type it as such. Please revert this.

Reply:
I did this, because dtslint wanted me to do that :/ Based on your next point 👇